### PR TITLE
Code Quality: Preview popup providers

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #issue...

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [x] Did you remove any strings from the en-us resource file?
   - [x] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Install QuickLook
   2. Select file and press space bar
   3. Use arrow keys to switch the preview
   4. Close QuickLook
   5. Press space bar and verify app doesn't crash

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

File: src/Files.App/ServicesImplementation/PreviewPopupService.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+using Files.Backend.Services;
+
+namespace Files.App.ServicesImplementation
+{
+	/// <inheritdoc cref="IPreviewPopupService"/>
+	internal sealed class PreviewPopupService : ObservableObject, IPreviewPopupService
+	{
+		public Task<IPreviewPopupProvider> GetProviderAsync()
+		{
+			return QuickLookProvider.Instance;
```
Comment (d2dyno1) [inline src/Files.App/ServicesImplementation/PreviewPopupService.cs:14]: Use `Task.FromResult()`

**Review by d2dyno1 — CHANGES_REQUESTED** [review]

File: src/Files.App/ServicesImplementation/PreviewPopupService.cs (hunk 1 above)
Comment (yaira2) [reply src/Files.App/ServicesImplementation/PreviewPopupService.cs]: Thank you

**Comment by d2dyno1** [comment]
Is `SeerProProvider` going to be implemented in this PR?

**Comment by yaira2** [comment]
This is just providing the infrastructure

**Pull request merged**